cips/cip-6-second-blocks.md

```diff
@@ -0,0 +1,72 @@
+| cip | TBD |
+| - | - |
+| title | Reduce block time to six seconds |
+| description | Proposal to reduce block time on celestia-app to 6 seconds, from 12. |
+| author | Josh Stein ([@jcstein](https://github.com/jcstein)), Rootul Patel ([@rootulp](https://github.com/rootulp))|
+| discussions-to | <https://forum.celestia.org/t/cip-decrease-block-time-to-6-seconds/1836> |
+| status | Draft |
+| type | Standards Track |
+| category | Core |
+| created | 2024-10-09 |
+
+## Abstract
+
+This CIP proposes to reduce the block time on celestia-app to 6 seconds, from 12. This will double the throughput and reduce the time it takes for transactions to be finalized in half. This CIP also proposes to increase the `ttl-num-blocks` parameter in the mempool configuration from 5 to 12 to maintain consistency with the new block time.
+
+## Motivation
+
+The motivation for this CIP stems from a discussion in Core Devs Call 17, where it was proposed to reduce the block time to 6 seconds from 12 seconds.
+
+## Specification
```

Comment:
I think this specification section should include a short blurb on why `TimeoutCommit` and `TimeoutPropose` were moved from local config parameters into versioned parameters controlled by the state machine. cc: @staheri14

Reply:
using this for now "The `TimeoutCommit` and `TimeoutPropose` parameters were moved from local config parameters into versioned parameters controlled by the state machine to ensure consistency and correctness across different protocol versions. This change allows the system to adapt dynamically to version-specific requirements, improving overall stability and interoperability during upgrades."

Reply:
https://github.com/celestiaorg/CIPs/pull/216/commits/837357a1aa8e705fb11aff72c3b6ace03b225382